client/my-sites/themes/thanks-modal.jsx

```diff
@@ -92,17 +87,16 @@ const ThanksModal = React.createClass( {
 		);
 	},
 
-	renderWporgThemeInfo( themeUri ) {
-		if ( themeUri ) {
+	renderThemeInfo() {
+		const { detailsUrl } = this.props;
+		if ( detailsUrl ) {
 			return (
-				<li>
-					{ translate( 'Learn more about this {{a}}awesome theme{{/a}}.', {
-						components: {
-							a: <a href={ themeUri }
-								onClick={ this.onLinkClick( 'org theme' ) } />
-						}
-					} ) }
-				</li>
+				translate( '{{a}}Learn more about{{/a}} this theme.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Learn more about this {{a}}awesome theme{{/a}}.' )` *ES Score: 6.35*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).